traiders/backend/api/signal_handlers/order_signals.py

```diff
@@ -0,0 +1,74 @@
+import logging
+from django.dispatch.dispatcher import receiver
+from django.db.models.signals import post_save
+from django.db import transaction
+from ..models import BuyOrder, Parity, StopLossOrder, Asset, OnlineInvestment
+
+logger = logging.getLogger(__name__)
+
+
+@receiver(post_save, sender=Parity)
+@transaction.atomic
+def handle_orders(sender, instance: Parity, created, **kwargs):
+    logger.info(f'Handling orders after saving a parity for: {instance}')
+
+    base_eq = instance.base_equipment
+    target_eq = instance.target_equipment
+    ratio = instance.close
+
+    buy_orders = BuyOrder.objects.filter(base_equipment=base_eq,
+                                         target_equipment=target_eq)
+
+    stoploss_orders = StopLossOrder.objects.filter(base_equipment=base_eq,
+                                                   target_equipment=target_eq)
+
+    for order in buy_orders:
+        if order.buy_ratio > ratio:
```

Comment:
To be more efficient, you can also filter by ratio (buy_ratio__gt) and omit this check. This way we wont process all of the orders each time a new parity is added.